mkdocs.yml

```
@@ -936,7 +936,7 @@ extra:
   latest_tag_4_5: '4.5.7'
   latest_tag_4_6: '4.6.16'
 
-  symfony_doc: 'https://symfony.com/doc/5.4'
+  symfony_doc: 'https://symfony.com/doc/5.x'
   user_doc: 'https://doc.ibexa.co/projects/userguide/en/master'
```

Side note: since master is hidden, maybe we should make this change: 
```suggestion
  user_doc: 'https://doc.ibexa.co/projects/userguide/en/latest'
```

---

Definitely, nice catch!

---

Or maybe it should be set per branch?
  user_doc: 'https://doc.ibexa.co/projects/userguide/en/4.6' for 4.6
  user_doc: 'https://doc.ibexa.co/projects/userguide/en/3.3' for 3.3

---

We culd... but then we'd have to change all links when bumping version and remember about changing them when cherrypicking changes to older branches

---

We wouldn't have to do this - we could just use the variable, as we currently do.

The value of the variable would change between branches, but the syntax wouldn't.

This example:
```
[I'm a link]([[= user_doc =]]/personalization/personalization/)
```

would link to:
- https://doc.ibexa.co/projects/userguide/en/3.3/personalization/personalization/ on 3.3 branch of the dev doc
- https://doc.ibexa.co/projects/userguide/en/4.6personalization/personalization/ on 4.6 branch of the dev doc

So, no changing links when creating the 5.0 branch (and no changes when cherry-picking), we would only make one single change in mkdocs.yaml when creating the 5.0 branch - and the links would become "version-aware"

---

I'm suggesting we do the same thing we already do for the Symfony doc:
- https://github.com/ibexa/documentation-developer/blob/2.5/mkdocs.yml#L411
- https://github.com/ibexa/documentation-developer/blob/4.6/mkdocs.yml#L938

which is not causing any issues currently

---

- #2618
- #2619

---

@mnocon @dabrt I propose to not make decision about this in this PR. Let dev's master targets user's master for now. I'll test and we'll discuss latter what 4.6 targeting 4.6 and 3.3 targeting 3.3 imply.